src/lib/y2network/autoinst_profile/interface_section.rb

```diff
@@ -260,9 +261,16 @@ def init_from_hashes(hash)
         hash = rename_key(hash, "bridge_forwarddelay", "bridge_forward_delay")
         super(hash)
 
-        if hash["aliases"]
-          self.aliases = hash["aliases"].values.map { |h| AliasSection.new_from_hashes(h) }
+        # When the name and the device attributes are given then the pre network-ng behavior will be
+        # adopted using the name as the description and the device as the name (bsc#1192270).
+        unless (hash.fetch("name", "").empty? || hash.fetch("device", "").empty?)
```

Comment:
I would need to double check but in case of SLE-15-SP2 I think hash["name"] will be "" if not given and the same with hash["device"] I think it changed in SLE-15-SP3. So, I was a little bit conservative... and tried to make it work in both cases with expected nil or empty values.

Reply:
BTW would be nice if @imobachgs can check it just to know his opinion about adding the description element in SP2.

Reply:
What about this version?

~~~
values = hash.fetch_values("name", "device")

if values.none?(&:nil?) && values.none?(&:empty?)
  ...
end
~~~